docs/en/getting-started/local_quickstart.md

```diff
@@ -258,7 +259,7 @@ pip install google-genai
 {{< /tab >}}
 {{< tab header="ADK" lang="bash" >}}
 
-pip install google-adk langchain
```

Comment:
langchain is required. This was confirmed by a CE to update https://google.github.io/adk-docs/tools/google-cloud-tools/#configure-and-deploy

Reply:
Sure, will remove that.